Northstar.Client/mod/scripts/vscripts/ui/menu_ns_serverbrowser.nut

```diff
@@ -90,7 +90,39 @@ struct {
 	array<var> serversRegion
 } file
 
+enum eProgressType
+{
+    DOWNLOAD = 0,
+    EXTRACTION = 1
+}
 
+// This struct holds information about the currently downloaded mod:
+struct ModDownloadProgress {
+	float received					// received quantity
+	float total						// total quantity expected
+	float receivedPercentage		// ratio of received / expected (computed in native for PERFS!)
+	int progressType				// nature of previous data (0=download stats [in MBs], 1=extraction stats [in files count])
+	
+	// The following information should only be used with big files (as there's no need to display this
+	// for small files due to extraction speed):
+	float currentFileExtracted		// extracted size of current file
+	float currentFileTotal 			// total size of current file
+}
+
+/**
+ * Builds up a ModDownloadProgress struct from values retrieved from native code.
+ **/
+ModDownloadProgress function build(array<float> values)
```

Comment:
Function names should begin with an uppercase letter

Reply:
Addressed in 2bee48390b284242cfdb6cf846621cab3b99e7bc.